### ISSUE TITLE
battery status

### ISSUE THREAD

**Issue by cybernard** [Python-roborock/python-roborock#95]
My phone roborock app displays the battery percentage is there any chance at getting this data?

**Comment by humbertogontijo** [comment]
Yes. Look at field battery from the response of get_status command
